src/bus_constants/39.json

```
@@ -0,0 +1,150 @@
+            {
+                "stop_name": "Jamaica Plain Center",
+                "branches": null,
+                "station": "jpctr",
+                "order": 2,
+                "stops": {
+                    "1": [
+                        "39-1-1128"
+                    ],
+                    "0": [
+                        "39-0-1939"
+                    ]
+                }
+            },
+            {
+                "stop_name": "Centre St & S Huntington",
+                "branches": null,
+                "station": "shunt",
+                "order": 3,
+                "stops": {
+                    "1": [
+                        "39-1-11131"
+                    ],
+                    "0": [
+                        "39-0-1160"
+                    ]
+                }
+            },
+            {
+                "stop_name": "Heath Street Loop",
+                "branches": null,
+                "station": "heath",
+                "order": 4,
+                "stops": {
+                    "1": [
+                        "39-1-65741",
+                        "39-1-6574"
+                    ],
+                    "0": [
+                        "39-0-31365"
+                    ]
+                }
+            },
+            {
+                "stop_name": "Brigham Circle",
+                "branches": null,
+                "station": "brghm",
+                "order": 5,
+                "stops": {
+                    "1": [
+                        "39-1-1317"
+                    ],
+                    "0": [
+                        "39-0-1363"
+                    ]
+                }
+            },
+            {
+                "stop_name": "Huntington Ave & Longwood Ave",
+                "branches": null,
+                "station": "hlong",
+                "order": 6,
+                "stops": {
+                    "1": [
+                        "39-1-31317"
+                    ],
+                    "0": [
+                        "39-0-91391"
+                    ]
+                }
+            },
+            {
+                "stop_name": "Northeastern University",
+                "branches": null,
+                "station": "nuniv",
+                "order": 7,
+                "stops": {
+                    "1": [
+                        "39-1-81317"
+                    ],
+                    "0": [
+                        "39-0-41391"
+                    ]
+                }
+            },
+            {
+                "stop_name": "Huntington Ave & Belvidere St",
+                "branches": null,
+                "station": "hunbv",
+                "order": 8,
+                "stops": {
+                    "1": [
+                        "39-1-11389"
+                    ],
+                    "0": [
+                        "39-0-11388"
+                    ]
+                }
+            },
+            {
+                "stop_name": "Copley",
```

```suggestion
                "stop_name": "Copley (Inbound)",
```
Since only inbound buses stop here.
2 other points:
We're losing legacy data from bdal by not including it, and hunbv won't have inbound data before June 2020.
I think this is the correct treatment for now, but I'm going to open an issue in case we figure out a way to cleanly solve this later.
(from http://roster.transithistory.org/MBTARouteHistory.pdf : "In June 2020 the inbound route was changed to use Ring Road in place of Belvidere and Dalton Street between Huntington Ave. and Boylston St.")